src/us/kbase/typedobj/test/files/SubsetAndMetadataExtraction/KB.spec

```diff
@@ -53,65 +48,9 @@ module KB {
 	
 	
 	/*
-	  @searchable ws_subset names
-	  @searchable ws_subset numbers.[*]
-	*/
-	typedef structure {
-		list<string> names;
-		list<int> numbers;
-		string throwAwayData;
-	} ListStruct;
-
-
-	/*
-	  @searchable ws_subset keys_of(crazydata.*.*)
-	*/
-	typedef structure {
-		mapping<string,mapping<string,mapping<string,int>>> crazydata;
-	} DeepMaps;
-	
-	/*
-	  @searchable ws_subset keys_of(d1) keys_of(d2) keys_of(d3)
-	  @searchable ws_subset keys_of(d4) keys_of(d5) keys_of(d6)
-	*/
-	typedef structure {
-		mapping<string,mapping<string,int>> d1;
-		mapping<string,list<float>> d2;
-		mapping<string,tuple<int,int>> d3;
-		mapping<string,string> d4;
-		mapping<string,int> d5;
-		mapping<string,float> d6;
-	} KeysTest;
-
-
-	/*
-	@searchable ws_subset name
-	*/
-	typedef structure {
-		string name;
-		string seq;
-		int length;
-	} Subdata;
-
-	/*
-	@optional dl dm dml
-	@searchable ws_subset d.(name,length)
-	@searchable ws_subset dl.[*].name
-	@searchable ws_subset dm.*.(seq,length)
-	@searchable ws_subset dml.*.[*].name
-	*/
-	typedef structure {
-		Subdata d;
-		list<Subdata> dl;
-		mapping<string,Subdata> dm;
-		mapping<string,list<Subdata>> dml;
-	} NestedData;
-	
-	/*
-	searchable ws_subset s i
 	@metadata ws s As String Data
 	@metadata ws i As Integer
-	@metadata ws	f	as floater
```

Comment:
Not related to subdata extraction, but I vaguely recall that extra spaces here was for testing the parser, so we should keep them.